Zilon.Core/Zilon.Core/World/Globe.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Zilon.Core.Tactics;
+using Zilon.Core.Tactics.Behaviour;
+
+namespace Zilon.Core.World
+{
+    /// <summary>
+    /// Globe base implementation.
+    /// </summary>
+    public sealed class Globe : IGlobe
+    {
+        private int _turnCounter;
+
+        private readonly IList<ISectorNode> _sectorNodes;
+
+        private readonly ConcurrentDictionary<IActor, TaskState> _taskDict;
+        private readonly IGlobeTransitionHandler _globeTransitionHandler;
+
+        public IEnumerable<ISectorNode> SectorNodes { get => _sectorNodes; }
+
+        public Globe(IGlobeTransitionHandler globeTransitionHandler)
+        {
+            _taskDict = new ConcurrentDictionary<IActor, TaskState>();
+
+            _sectorNodes = new List<ISectorNode>();
+
+            _globeTransitionHandler = globeTransitionHandler ?? throw new ArgumentNullException(nameof(globeTransitionHandler));
+        }
+
+        public void AddSectorNode(ISectorNode sectorNode)
+        {
+            if (sectorNode is null)
+            {
+                throw new ArgumentNullException(nameof(sectorNode));
+            }
+
+            _sectorNodes.Add(sectorNode);
+            sectorNode.Sector.TrasitionUsed += Sector_TrasitionUsed;
+            sectorNode.Sector.ActorManager.Removed += ActorManager_Removed;
+        }
+
+        private void ActorManager_Removed(object sender, ManagerItemsChangedEventArgs<IActor> e)
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'sender'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)